configure.ac

```diff
@@ -200,25 +200,22 @@ AC_ARG_WITH([cuse],
 )
 
 if test "$with_cuse" != "no"; then
-    LIBFUSE_CFLAGS=$(pkg-config fuse --cflags 2>/dev/null)
-    if test $? -ne 0; then
-        if test "$with_cuse" = "yes"; then
-            AC_MSG_ERROR("Is fuse-devel installed? -- could not get cflags for libfuse")
-        else
-            with_cuse=no
-        fi
-    else
-        with_cuse=yes
-    fi
+	if pkg-config fuse3; then
+		LIBFUSE_CFLAGS=$(pkg-config fuse3 --cflags)
+		LIBFUSE_LIBS=$(pkg-config fuse3 --libs)
+		with_cuse=yes
+	elif pkg-config fuse; then
+		LIBFUSE_CFLAGS="$(pkg-config fuse --cflags) -DWITH_FUSE2"
+		LIBFUSE_LIBS=$(pkg-config fuse --libs)
+		with_cuse=yes
+	else
+		AC_MSG_ERROR("Is fuse3-devel or fuse-devel installed? -- could not found libfuse3 or libfuse")
+	fi
 fi
 
 dnl with_cuse is now yes or no
-if test "$with_cuse" != "no"; then
-    LIBFUSE_LIBS=$(pkg-config fuse --libs)
-    if test $? -ne 0; then
-        AC_MSG_ERROR("Is fuse-devel installed? -- could not get libs for libfuse")
-    fi
-    AC_SUBST([LIBFUSE_CFLAGS])
+if test "$with_cuse" == "yes"; then
+	AC_SUBST([LIBFUSE_CFLAGS])
```

Comment:
Use a single '=' in comparisons in shell scripts.
Indentation of AC_SUBST with spaces.
And please squash the two patches into one. Keep the description and title of the first.

Reply:
ooops
a typo ...